### ISSUE TITLE
any_urlfield integratin is broken

### ISSUE THREAD

**Issue by bashu** [django-fluent/django-fluent-pages#134]
see screenshot, but no errors in a console 

![screen shot 2018-02-21 at 15 21 40](https://user-images.githubusercontent.com/144783/36469766-f4fea38e-171a-11e8-80ff-2fce387f51a5.png)

pip freeze output:
```
Django==1.11.10
django-any-imagefield==0.9
django-any-urlfield==2.4.2
django-contrib-comments==1.8.0
django-fluent-contents==1.2.2
django-fluent-pages==1.1.2
django-fluent-utils==2.0
django-js-asset==1.0.0
django-mptt==0.9.0
django-parler==1.9.2
django-polymorphic==2.0.2
django-polymorphic-tree==1.5
django-slug-preview==1.0.4
django-suit==0.2.25
django-tag-parser==3.1
django-template-analyzer==1.6.1
django-tinymce==2.7.0
django-wysiwyg==0.8.0
docutils==0.14
future==0.16.0
html5lib==1.0.1
locustio==0.8.1
Markdown==2.6.11
micawber==0.3.5
msgpack-python==0.5.4
Pillow==5.0.0
Pygments==2.2.0
pytz==2018.3
six==1.11.0
South==1.0.2
textile==3.0.0
webencodings==0.5.1
```

to reproduce just ``git clone https://github.com/django-fluent/django-fluent-pages.git`` and try 'example' project with installed and configured 'django-any-urlfield'

**Comment by bashu** [comment]
@vdboor not sure if this project or https://github.com/edoburu/django-any-urlfield/ is right place

**Comment by bashu** [comment]
@vdboor maybe I am stupid, but it turns out that the page you want to select, *must* be published, otherwise you will get an error message like above. Maybe the  "page chooser" should be limited to published pages only or error message should be more descriptive

**Comment by bashu** [comment]
@vdboor what if I decide to "unpublish" previously published and selected page? for example I can not save "redirect" page without selecting any other published page

**Comment by vdboor** [comment]
@bashu Do I understand it correctly that this happens when a page is unpublished?

This is the right package for the bug. I had to check it too, but in `fluent_pages/models/__init__.py`, the registration is done to any-urlfield and the limit to published pages happen in the `PageChoiceField` in `fluent_pages/forms/fields.py`.

**Comment by vdboor** [comment]
Fixed in v2.0.4!
